### DISCUSSION TITLE
API not working properly

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#6650] [Error / Bug Report] [answered]
I want to use the API. In the playground I keep getting the same error message 

![2023-08-13 15_34_26-Playground - https___docs cocolotravel com_graphql](https://github.com/requarks/wiki/assets/103907166/6f968e7b-994d-4ebb-baf5-a23198c8f755)

Any idea why this could be ?

**Comment by NGPixel** [comment, marked as answer]
The answer is literally in the results panel on the right... You're using an "orderBy" argument which doesn't exist and this is not how you query pages.

There's an example on [how to query pages in the docs](https://docs.requarks.io/dev/api#fetch-all-pages).

**Comment by ghost** [comment]
I literally copy pasted the query example in the doc as you can see in the screenshot
![image](https://github.com/requarks/wiki/assets/103907166/54760f34-de40-4352-bbd0-5be4ec359bc1)

**Reply by NGPixel** [reply]
No you did not. Reread what you typed.

**Comment by ghost** [comment]
Sorry @NGPixel I see what happened. Please could you tell the difference between 

https://next.js.wiki/docs/dev/api.html

and 

https://docs.requarks.io/dev/api#fetch-all-pages

Cuz that confused me as this is where I was looking for documentation.

**Reply by NGPixel** [reply]
next.js.wiki is the upcoming v3 release (see the red warning at the top of the page)

docs.requarks.io is the current stable version.
